StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/PaymentMethodRowButton.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  PaymentMethodRowButton.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 5/9/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+protocol PaymentMethodRowButtonDelegate: AnyObject {
+    func didSelectButton(_ button: PaymentMethodRowButton)
+    // TODO(porter) Add did delete and did update
+}
+
+final class PaymentMethodRowButton: UIView {
+
+    struct ViewModel {
+        let appearance: PaymentSheet.Appearance
+        let text: String
+        let image: UIImage
+        // TODO(porter) Add can remove and can update
+    }
```

Comment:
Probably not totally needed right now, but I anticipate this will scale better and am hoping in the future to create view models for `PaymentMethodRowButton` from a `STPPaymentMethod`. Mainly trying to keep the size of `PaymentMethodRowButton.init` down and offload the injection building somewhere else. Not totally needed but kinda like the direction.

```swift
extension STPPaymentMethod {
   var rowButtonViewModel: PaymentMethodRowButton.ViewModel {...}
}

// Somewhere else
let rowButtons = savedPaymentMethods.map{PaymentMethodRowButton(appearance: appearance, $0.rowButtonViewModel)}
```